spec/requests/portfolio_items_spec.rb

```diff
@@ -192,7 +192,7 @@
     let(:url) { "#{api}/portfolio_items/#{portfolio_item.id}/provider_control_parameters" }
 
     it "fetches plans" do
-      stub_request(:get, "http://localhost/api/topological-inventory/v1.0/sources/568/container_projects")
+      stub_request(:get, "http://localhost/api/topological-inventory/v2.0/sources/568/container_projects")
```

Comment:
@syncrou Does it make sense to have this as a function in the spec_helper called topo_url, which takes in something like the partial_href and returns the full url

topo_url(sources/568/container_projects) and returns http://localhost/api/topological-inventory/v1.0/sources/568/container_projects

Reply:
That's probably a good idea, we could even pass in the version (if necessary):

```
def topological_url(partial_path, api_version = "v2.0")
  "http://topology/api/topological-inventory/#{api_version}/#{partial_path}"
end
```

Reply:
@mkanoor, @eclarizio I was thinking.  This PR is green and it is missing all the nice helper changes I have in https://github.com/RedHatInsights/catalog-api/pull/518. I was thinking we merge this as is ( it fixes our versioning problem ) and then I add the `topo_url` changes into #518?

What do you think?

Reply:
Seems good to me